addons/mail/static/src/messaging/component/file_uploader/file_uploader_tests.js

```diff
@@ -72,7 +74,7 @@ QUnit.test('no conflicts between file uploaders', async function (assert) {
     );
     await nextAnimationFrame(); // we can't use afterNextRender as fileInput are display:none
     assert.strictEqual(
-        Object.keys(this.env.store.state.attachments).length,
+        this.env.entities.Attachment.all.length,
```

Comment:
I don't know how common it is to do `.all.length`, but if so maybe do a getter for it?

Reply:
quite rare, I think just 2 occurrences among all entities, so getter is not worth in my opinion